src/main/resources/CodyBundle.properties

```diff
@@ -169,3 +169,27 @@ settings.migration.llm-upgrade-notification.body=\
 
 # URLs
 url.sourcegraph.subscription=https://sourcegraph.com/cody/subscription
+
+# Authentication
+login.dialog.title=Add Sourcegraph Account
+login.dialog.authorize-in-browser=Authorize in Browser
+login.dialog.add-account=Add Account
+login.dialog.show-advanced=Show Advanced...
+login.dialog.hide-advanced=Hide Advanced...
+login.dialog.check-browser=Logging in, check your browser
+login.dialog.instance-url.label=Instance URL:
+login.dialog.instance-url.comment=Enter the address of your sourcegraph instance. For example https://sourcegraph.<yourcompany>.org
```

Comment:
Looks like IntelliJ parses HTML tags in these strings. Try replacing the angle brackets with `&lt;` and `&gt;`.